src/scss/mixins/_buttons.scss

```diff
@@ -0,0 +1,5 @@
+@mixin button($color: $color-basic, $bg-color: $color-primary, $text-transform: uppercase) {
+  color: $color;
+  background-color: $bg-color;
+  text-transform: $text-transform;
+}
```

Comment:
Can be improved by adding states on hover, active, disabled or focus

Reply:
I use these styles from Angular material